roles/base/tasks/opv.yml

```diff
@@ -61,6 +62,11 @@
     dest: /home/opv/.bashrc
     line: "source ~/venvs/opv/bin/activate"
 
+- name: Extract opv-status
+  unarchive:
+    src: opv-status.zip
```

Comment:
Compiled source code should not be commited anywhere ! And certainly not on the repository.
Remove it.
- Fetch the source from git (maybe fetch a release if you want instead of a branch).
- install node
- make an npm i on the sources directory and build to compile your project (with the good environment)

Take a look at some other npm web based projects and how they are deployed using Ansible, suff like this https://codelike.pro/deploy-nodejs-app-with-ansible-git-pm2/ (i'm sure you can find better exemples).